### DISCUSSION TITLE
is flux bootstrap supposed to fail when kustomize-controller is not part of the components list?

### DISCUSSION THREAD

**Discussion by jack-evans** [fluxcd/flux2#1057] [Q&A] [answered]
when I run`flux bootstrap github .... --components source-controller,helm-controller` I get as output:
```
✗ component kustomize-controller is required
```

Is there a reason why the kustomize-controller is required? We only use helm charts so would only really have need for the helm-controller and source-controller so to have something extraneous running seems unnecessary.

If the kustomize-controller is not required would the best way to install it be through `flux install` instead?

**Comment by stefanprodan** [comment, marked as answer]
kustomize-controller is the component that applies Flux’s CRDs, components, namespaces, GitRepositories, HelmRepositories and HelmReleases manifests from Git on your cluster. If you want to use Flux for Helm operations without GitOps then see the docs here: https://toolkit.fluxcd.io/faq/#can-i-use-flux-helmreleases-without-gitops
